tests/shutdown/svirt_upload_assets.pm

```diff
@@ -58,6 +58,10 @@ sub extract_assets {
 
     upload_asset("/var/lib/libvirt/images/$name", 1, 1);
     assert_screen('svirt-asset-upload-hdd-image-uploaded', 1000);
+
+    # clean up on s390pb
+    type_string("rm -f /var/lib/libvirt/images/$name && echo 'OK'\n");
```

Comment:
this code does not run on the SUT but on the hypervisor, so it has no access to the serial console of the SUT.